docs/eigenda/deployed-contracts.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 8
+title: Contracts
+---
+
+## EigenDA Contracts
+
+For more information on the contracts and their technical specifications, please see:
+[EigenLayer Middleware Docs](https://github.com/Layr-Labs/eigenlayer-middleware/tree/m2-mainnet/docs).
+
+### Current Mainnet Deployment
+
+No contracts have been deployed to mainnet yet.
+
+### Current Testnet Deployment
```

Comment:
Please add branch/release context like the text removed in the other doc :heart: